definitions/npm/react-native-simple-store_v1.x.x/flow_v0.25.x-/react-native-simple-store_v1.x.x.js

```diff
@@ -2,9 +2,14 @@ declare module 'react-native-simple-store' {
   declare type Key = string | Array<Array<any>>;
   declare type Value = string | boolean | Object | Array<any>;
 
-  declare function get(key: Key): Promise<?any>;
-  declare function save(key: Key, value: Value): Promise<?any>;
-  declare function update(key: string, value: Value): Promise<?any>;
-  declare function keys(): Promise<?any>;
-  declare function push(key: string, value: Value): Promise<?any>;
+  declare type SimpleStore = {
+    get: (key: Key)=> Promise<?any>,
```

Comment:
`?any` is weird, `any` alone is enough, since it bascially means no typing, so anything will pass.

Reply:
good point